### PR TITLE
8328642: Convert applet test MouseDraggedOutCauseScrollingTest.html to main

### PR DESCRIPTION
**Pull request by prrace**
Converts java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.html
to a main program.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328642](https://bugs.openjdk.org/browse/JDK-8328642): Convert applet test MouseDraggedOutCauseScrollingTest.html to main (**Bug** - P4)

### Reviewers
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - Committer) ⚠️ Review applies to [979e4857](https://git.openjdk.org/jdk/pull/18414/files/979e485703a2aebbcf1d5694a1aca61301736fa5)
 * [Alexander Zvegintsev](https://openjdk.org/census#azvegint) (@azvegint - **Reviewer**) ⚠️ Review applies to [979e4857](https://git.openjdk.org/jdk/pull/18414/files/979e485703a2aebbcf1d5694a1aca61301736fa5)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18414/head:pull/18414` \
`$ git checkout pull/18414`

Update a local copy of the PR: \
`$ git checkout pull/18414` \
`$ git pull https://git.openjdk.org/jdk.git pull/18414/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18414`

View PR using the GUI difftool: \
`$ git pr show -t 18414`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18414.diff">https://git.openjdk.org/jdk/pull/18414.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18414#issuecomment-2010684018)

### CODE REVIEW

**Review by kumarabhi006 — APPROVED** [review]
LGTM.

File: test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java [hunk 1]
```diff
@@ -22,29 +22,29 @@
...
-  test
+  @test
   @bug 6243382 8006070
   @summary Dragging of mouse outside of a List and Choice area don't work properly on XAWT
-  @author Dmitry.Cherepanov@SUN.COM area=awt.list
-  @run applet/manual=yesno MouseDraggedOutCauseScrollingTest.html
+  @requires (os.family == "linux")
+  @library /java/awt/regtesthelpers
+  @run main/manual MouseDraggedOutCauseScrollingTest
 */
```
Comment (azvegint) [inline test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java:31]: We usually move this and instructions block closer to a class declaration, for better readability.

**Review by azvegint — APPROVED** [review]

File: test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java [hunk 2]
```diff
@@ -59,188 +59,68 @@ public void init()
         for (int i = 1; i < 100; i++)
             multipleList.add(""+i);
 
-        this.add(choice);
-        this.add(singleList);
-        this.add(multipleList);
+        frame.add(choice);
+        frame.add(singleList);
+        frame.add(multipleList);
+        frame.setSize(400,100);
```
Comment (turbanoff) [inline test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java:65]:
```suggestion
        frame.setSize(400, 100);
```

File: test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java (hunk 1 above)
Comment (prrace) [reply test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java:31]: So people have been saying this, and I've even done it when asked, but I'm not sure I agree.
Although it has to go below the legal stuff, after that I like to see the test tag right at the top of the file,
not below a random number of imports.

**Pull request closed without merging**